### DISCUSSION TITLE
chronological order of nodes (by time)

### DISCUSSION THREAD

**Discussion by gephiforuniversity** [gephi/gephi#2768] [Q&A]
Hi,
I have a problem with the time representation in gephi. 

I have data about Different people moving between different places (the places are my nodes, less than 100). Data is with time of departure and time of arrival for each way, I need a directed graph. 

I want to have the chronological order of the nodes to visualize the pathways the people are going to find out about different paths and which paths are most relevant

i think i need timestamps in format YYYY/MM/DD hh:mm:ss. but it doesn´t work. i don´t need to know at what time a node is arrived, just the order and the weight of the edge to say how many people walk from node A to B and so on.

can you help me please? i need the solution for my studies. 

thanks in advance :)

**Comment by mbastian** [comment]
Hi @gephiforuniversity, can you let us know how you are importing your data in Gephi? Using GEXF format for instance?
